tests/base.py

```diff
@@ -134,6 +134,9 @@ def setUpClass(cls):
         cls._original_bcrypt_log_rounds = settings.BCRYPT_LOG_ROUNDS
         settings.BCRYPT_LOG_ROUNDS = 1
 
+        cls._original_enable_notification_subscription_creation = settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION
+        settings.ENABLE_NOTIFICATION_SUBSCRIPTION_CREATION = False
```

Comment:
There should also be a line in `tearDownClass` assigning this to the `_original` value